src/routes/(site)/blog/[id]/blog-body.svelte

```diff
@@ -1,5 +1,41 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
   export let data = '';
+  onMount(() => {
+    let pre = document.querySelectorAll('pre');
+    let svg = `<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"
+  ><rect
+    x="128"
+    y="128"
+    width="336"
+    height="336"
+    rx="57"
+    ry="57"
+    fill="none"
+    stroke="currentColor"
+    stroke-linejoin="round"
+    stroke-width="32"
+  /><path
+    d="M383.5 128l.5-24a56.16 56.16 0 00-56-56H112a64.19 64.19 0 00-64 64v216a56.16 56.16 0 0056 56h24"
+    fill="none"
+    stroke="currentColor"
+    stroke-linecap="round"
+    stroke-linejoin="round"
+    stroke-width="32"
+  /></svg
+>`;
+    var svgCode = encodeURI(svg);
+    for (let i = 0; i < pre.length; i++) {
+      let parentDiv = pre[i].parentElement;
+      parentDiv!.style.position = 'relative';
```

Comment:
do not resolve object possibly null with a non null assertion like this, add an if statement
<img width="538" alt="image" src="https://user-images.githubusercontent.com/8981287/196589423-80ab1537-2b5e-45ca-a776-738a9b2bda4d.png">

Reply:
also prefer const

Reply:
done